tamr_unify_client/models/dataset/resource.py

```diff
@@ -65,21 +67,62 @@ def update_records(self, records, **json_args):
         :returns: JSON response body from server.
         :rtype: :py:class:`dict`
         """
-
-        def _stringify_updates(updates):
-            for update in updates:
-                yield json.dumps(update, **json_args).encode("utf-8")
+        stringified_updates = (
```

Comment:
This breaks streaming :-(

Reply:
Even though this looks like a tuple-comprehension (which isn't a thing), it's a [generator expression](https://www.python.org/dev/peps/pep-0289/#the-details). So records are still streamed and never materialized in RAM.